### PR TITLE
❗BREAKING CHANGE❗️Ensure compatibility with graphql-php v15

### PR DESCRIPTION
**Pull request by mfn**
## Summary
See https://github.com/webonyx/graphql-php/discussions/1231

❗ Merging this will require a major version bump due to all around incompatible changes forced by graphql-php ❗ 

### TODO
- [x] Figure out how to fix or workaround the disabled test `\Rebing\GraphQL\Tests\Unit\ConfigTest::testSecurity`
- [x] Wait for official v15 release and remove hack commit "graphql-php-v15: workaround to allow testing with graphql-php v15"
- [x] _Then_ make sure integration tests are green
  They current aren't due to the composer.json hack

---

Type of change:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Misc. change (internal, infrastructure, maintenance, etc.)

Checklist:
- [x] Existing tests have been adapted and/or new tests have been added
- [x] Add a CHANGELOG.md entry
- [x] Update the README.md
- [x] Code style has been fixed via `composer fix-style`

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -38,9 +38,9 @@
         "ext-json": "*",
         "illuminate/contracts": "^6.0|^8.0|^9.0",
         "illuminate/support": "^6.0|^8.0|^9.0",
-        "laragraph/utils": "^1",
+        "laragraph/utils": "1.4.1",
```
Comment (spawnia) [inline composer.json:41]:
```suggestion
        "laragraph/utils": "dev-graphql-php-15",
```

Should be the easier temporary fix.

**Comment by leonardocustodio** [comment]
We are having this issue when running tests on PHP 8.2, I guess because rebing is still using the older webonyx lib?

`PHP Deprecated:  Using ${var} in strings is deprecated, use {$var} instead in /vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 182`

**Review by tm1000 — APPROVED** [review]

**Comment by websitevirtuoso** [comment]
v15 has been released
https://github.com/webonyx/graphql-php/releases
Can we check this PR one more time please?
Let me know If I can assist in this

**Comment by sforward** [comment]
I have been testing these changes against our tests. I'm running into an exception when printing the generated schema via the upstream GraphQL\Utils\SchemaPrinter::doPrint. I've made a test case and a solution, but not sure if that's the best solution:

https://github.com/rebing/graphql-laravel/commit/0fc11f6fbc2a79513a9493c1aee7971560c54a2c

**Comment by sforward** [comment]
Sure, I've created a MR with the test!

**Pull request merged**